final_prep/mod_zero_hero.rb

```diff
@@ -56,16 +80,45 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  "name" => "Bob Barker",
+  "smell" => "Bengay and Aspercreme",
+  "weight" => 189,
+  "citiesDestroyed" => ["Tokyo", "Los Angeles", "San Fransisco"],
+  "luckyNumbers" => [76, 32, 1],
+  "address" => {number:"123", street:"Fake Street", state: "CO", zip: "80120"}
```

Comment:
The `address` hash would be a little easier to read if it were formatted with the various key-value pairs on separate lines.